internal/cli/cmds.go

```diff
@@ -464,7 +464,7 @@ func runList(language string, all bool, outputFormat outputFormat) {
 		var results map[api.PkgName]api.PkgSpec = nil
 		fileExists := util.Exists(b.Specfile)
 		if fileExists {
-			results = b.ListSpecfile()
+			results = b.ListSpecfile(false)
```

Comment:
maybe this should be true if the `-a` flag is specified?

Reply:
This is getting into the realm of "proper support for dependency groups", which isn't in scope for this change. I'm happy to make this `true`, though I don't feel great about a small scoped internal change leaking this much into the user's surface area.

Reply:
happy to stamp but still think it makes sense when you're asking for _all_ dependencies

Reply:
Reasonable point.

Reply:
```suggestion
			results = b.ListSpecfile(true)
```